packages/@aws-cdk/aws-apigateway/rosetta/default.ts-fixture

```diff
@@ -1,6 +1,6 @@
 // Fixture with packages imported, but nothing else
 import { Construct } from 'constructs';
-import { Stack } from '@aws-cdk/core';
+import { Duration, Stack } from '@aws-cdk/core';
```

Comment:
Why did we add the `Duration` import here?

Reply:
It *seems* to be necessary based on the changes I also made to `README.md`. I got a build failure without it since jsii-rosetta couldn't find `Duration` (if I interpreted the error message correctly). I guess it might be possible to choose another possible integration example but that might still require another import.